### PR TITLE
Add onMutation to watch for children updates.

### PR DESCRIPTION
**Pull request by rickymarcon**
Adding onMutation to watch for children updates in the feature-carousel element.

### CODE REVIEW

File: 0.5/elements/feature-carousel.html [hunk 1]
```diff
@@ -28,12 +28,18 @@
       interval: null, // num milliseconds to cycle panels.
       selected: 0,
       domReady: function() {
-        // TODO: use onMutation to watch for children updates.
         this.panels = Array.prototype.slice.call(
```
Comment (ebidel) [inline 0.5/elements/feature-carousel.html]: Why don't we just do:

    this.childrenUpdated();

That sets `this.panels` and the mutation observer

File: 0.8/elements/feature-carousel.html [hunk 2]
```diff
@@ -28,12 +28,18 @@
       interval: null, // num milliseconds to cycle panels.
       selected: 0,
       domReady: function() {
-        // TODO: use onMutation to watch for children updates.
         this.panels = Array.prototype.slice.call(
             this.$.content.getDistributedNodes());
 
+        this.onMutation(this, this.childrenUpdated);
```
Comment (ebidel) [inline 0.8/elements/feature-carousel.html]: same

File: 0.5/elements/feature-carousel.html (hunk 1 above)
Comment (rickymarcon) [inline 0.5/elements/feature-carousel.html]: Makes sense. Added.

**Comment by ebidel** [comment]
:+1:

**Pull request merged**